apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/FDADrugCodeDisplayLookupIT.java

```diff
@@ -0,0 +1,20 @@
+package gov.cms.bfd.server.war;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import gov.cms.bfd.data.fda.lookup.FdaDrugCodeDisplayLookup;
+import java.util.Optional;
+import org.junit.jupiter.api.Test;
+
+/** Verifies it can get an actual fda drug code display */
+public class FDADrugCodeDisplayLookupIT {
+  /** Verifies a real drug code display. */
+  @Test
+  public void verifiedRealDrugCode() {
+    FdaDrugCodeDisplayLookup fdaDrugCodeDisplayLookup =
+        FdaDrugCodeDisplayLookup.createDrugCodeLookupForProduction();
+    String fdaCodeDisplay =
```

Comment:
Should we assert not null on the fdaDrugCodeDisplayLookup; obviously an NPE could be thrown, which ultimately fails the test...it's just scary to see stacktraces, etc.